.github/workflows/python-app.yml

```diff
@@ -30,12 +30,19 @@ jobs:
     - name: Install pip dependencies
       run: |
         python -m pip install --upgrade pip setuptools
-        pip install flake8
+        pip install flake8 isort
     - name: Lint with flake8
       run: |
         # stop the build if there are Python syntax errors or undefined names
         flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
         # The GitHub editor is 127 chars wide
         flake8 . --count --max-complexity=10 --max-line-length=127 --statistics
+    - name: Test the imports with isort
+      run: |
+        # try to check every Python file with isort,
+        # those inside folders:
+        isort **/*.py --check-only -v
+        # but also outside the folders:
+        isort *.py --check-only -v
```

Comment:
I had to run `isort **/*.py` and `isort *.py` to cover _all the files_ from the repository.